runconfig/config.go

```diff
@@ -20,6 +20,7 @@ type Config struct {
 	Hostname        string                // Hostname
 	Domainname      string                // Domainname
 	User            string                // User that will run the command(s) inside the container
+	ArgsEscaped     bool                  `json:",omitempty"` // True if command is already escaped
```

Comment:
Does this really live in Config? Is it needed to be committed to the final image?